### PR TITLE
Feature/wbtech 186 input field result page UI

### PR DESCRIPTION
**Pull request by priyabyanjankar**
Changes made are 
added input field and result div in UI

### CODE REVIEW

**Review by ShrayaJoshi — APPROVED** [review]

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (amirsaga) [inline src/App.css:1]: Do we need this file?

File: src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```
Comment (amirsaga) [inline src/App.test.js:1]: This file ?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Deposit Calculator</title>
```
[the remark this answers is not in the document]
Comment (priyabyanjankar) [reply public/index.html]: Its done @manishbasnetnp

File: src/utils/depositFormOptions.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    foreign_buyer: 'false',
+    no_of_dependent: '0',
+    vic_property_location: 'Alpine Shire Council',
+    wa_property_location: 'South of the 26th parallel (from Kalbarri 6536)',
+    eligible_pensioner: 'false',
+    payment_method: 'PAPER_TRANSACTION',
+    total_income: 80000
+};
+
+const propertyStateOption = [
```
[the remark this answers is not in the document]
Comment (priyabyanjankar) [reply src/utils/depositFormOptions.js:22]: will look into it as changes to be made in UI-helper

**Review by manishbasnetnp — APPROVED** [review]

**Pull request merged**